packages/kit/test/apps/basics/test/client.test.js

```diff
@@ -412,9 +412,15 @@ test.describe('Page options', () => {
 		await Promise.all([page.waitForNavigation(), page.click('[href="/no-router/b"]')]);
 		expect(await page.textContent('button')).toBe('clicks: 0');
 
+		// wait until hydration before interacting with button
+		await page.waitForSelector('body.started');
+
 		await page.click('button');
 		expect(await page.textContent('button')).toBe('clicks: 1');
 
+		// wait until hydration before attempting backwards client-side navigation
+		await page.waitForSelector('body.started');
```

Comment:
Why would we need to do this again? Didn't it already happen with the line you added above?